Toolset/palettes/message box/behaviors/revmessageboxsinglelinemessagebehavior.livecodescript

```diff
@@ -22,8 +22,13 @@ on enterInField
    returnInField
 end enterInField
 
+local sColorizeMsg
+
 on textChanged
-   dispatch "revSEColorizeField" to stack "revSEUtilities" with the long id of me
+   cancel sColorizeMsg
+   get the long id of me
+   send "revSEColorizeField it" to stack "revSEUtilities" in 100 milliseconds
+   put the result into sColorizeMsg
```

Comment:
I wonder if it would be a good idea to make this a public handler in revSEUtilities and for it to store an array of colorize messages parameterized by the long id. Then we could also have the `100 milliseconds` bit as a constant in that script and people would be able to use it to colorize their fields without having to set up a send/cancel scenario

Reply:
(By no means essential to this patch ofc)

Reply:
I was actually considering making the single line msg box use the common editor behavior rather than do this but it would have involved significant enough changes that it would be better to do in develop.